### PR TITLE
Remove blog and Launchpad translation subpages

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Remove Blog translations and Launchpad translations subpages from the Translation Program page
Remove What we're working on section from the Translation Program page

## Related Issue

No related issue

### CODE REVIEW

File: src/content/contributing/translation-program/index.md
Comment (nhsz) [inline src/content/contributing/translation-program/index.md:1]: we should also remove all the links to `/contributing/translation-program/`

File: src/content/contributing/translation-program/launchpad-translations/index.md
Comment (nhsz) [inline src/content/contributing/translation-program/launchpad-translations/index.md:1]: we should also remove all the links to `/contributing/translation-program/launchpad-translations/`

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks! We should also remove the links to the removed pages to avoid having dead links

**Comment by lukassim** [comment]
> Thanks! We should also remove the links to the removed pages to avoid having dead links

For sure, thanks for raising this.
The links to these subpages from the main Translation Program page have been removed, and as far as I can see, we only link out to them on the content buckets page, which needs to be updated, and the translated versions of the Translation Program page.

Anywhere else we should remove these links from?

**Review by nhsz — APPROVED** [review]
LGTM

**Pull request merged**